subsys/caf/modules/sensor_manager.c

```diff
@@ -523,6 +523,23 @@ static bool app_event_handler(const struct app_event_header *aeh)
 		return false;
 	}
 
+	if (is_set_sensor_period_event(aeh)) {
```

Comment:
this event will not happen often - move it behind sensor_event.

Please also hide the code behind a function. If possible do the same for sensor event.